CMakeLists.txt

```diff
@@ -36,7 +36,10 @@ include_directories("include")
 
 add_subdirectory (src)
 add_subdirectory (examples)
-add_subdirectory (tests)
+
+if (NOT WIN32)
```

Comment:
Add a comment explaining why tests are disabled on windows

Reply:
> Add a comment explaining why tests are disabled ~on windows~

I'm partial to changing this from a Windows thing to all systems. If you don't have the tests pre-installed on _any_ system, then you can't actually run them.

Comment still valid, just needs to be broadened.